package.json

```diff
@@ -9,6 +9,7 @@
   "main": "./lib/api.js",
   "scripts": {
     "test": "node Makefile.js test",
+    "test:cli": "./node_modules/.bin/mocha",
```

Comment:
Would it work to simply say "mocha" here?

Reply:
It takes me quite a while to think about the naming and I can't think of a good one.🤦‍♂️
Maybe `mocha` works better.